rmw_zenoh_cpp/src/rmw_event.cpp

```diff
@@ -175,6 +175,13 @@ rmw_event_set_callback(
   return RMW_RET_OK;
 }
 
+///=============================================================================
+bool
+rmw_event_type_is_supported(rmw_event_type_t rmw_event_type)
+{
+  return rmw_zenoh_cpp::event_check_compatible(rmw_event_type);
```

Comment:
We don't need a new `event_check_compatible` method.
```suggestion
  return (rmw_zenoh_cpp::zenoh_event_from_rmw_event(rmw_event_type) != rmw_zenoh_cpp::ZENOH_EVENT_INVALID);
```